src/api/servers.conf.erb

```diff
@@ -0,0 +1,38 @@
+server {
+    listen <%= ENV['PORT'] %>;
+    server_name localhost;
+    charset utf-8;
+
+    location / {
+      proxy_pass http://localhost:<%= ENV['QUALICHARGE_API_PORT'] %>;
+      proxy_set_header Host $host ;
+      proxy_set_header X-Real-IP $remote_addr ;
+      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for ;
+      proxy_redirect off;
+
+      #
+      # Per endpoint rate-limits
+      #
+      # -- auth --
+      location = /api/v1/auth/whoami {
+        limit_req zone=auth rate=1r/s burst=2 nodelay;
```

Comment:
We should had a limit_req_status to raise a 429 HTTP code which is more explicit than the 503 default:


```suggestion
        limit_req_status 429;
```

References:

1. https://nginx.org/en/docs/http/ngx_http_limit_req_module.html
2. https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/503

Reply:
Should be applied to all endpoints.